agent_install.ps1

```diff
@@ -16,4 +16,20 @@ if (-not (Get-Service 'GCEAgent' -ErrorAction SilentlyContinue)) {
   New-Service -Name 'GCEAgent' -BinaryPathName 'C:\Program Files\Google\Compute Engine\agent\GCEWindowsAgent.exe' -StartupType Automatic -Description 'Google Compute Engine Agent'
 }
 
+$config = 'C:\Program Files\Google\Compute Engine\instance_configs.cfg'
+if (-not (Test-Path $config)) {
+  @'
+# GCE Instance Configuration
+
+# See https://cloud.google.com/compute/docs/images#os-details for details
```

Comment:
Currently this link doesn't provide any information. Please make sure to have something tracking that documentation update.